### PR TITLE
[WIP]add /approve support for ci-bot merge

### LINKED ISSUE #67: Auto automerge bot
Like https://github.com/pascalgn/automerge-action, or istio bot.

### PR DESCRIPTION
**Pull request by pacoxu**
Fixes #67

Comment `/merge` in later PR, it will be merged.

You have to manage the APPROVERS list here. (The current version of [gh-ci-bot](https://github.com/wzshiming/gh-ci-bot/) cannot read from https://github.com/merbridge/merbridge/blob/059c65629628b6b4df73dc2e02038cc1c7b9ef57/CODEOWNERS 😄)

### CODE REVIEW

**Comment by pacoxu** [comment]
test example:  https://github.com/pacoxu/merbridge/pull/1

File: .github/workflows/ci-bot.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - created
+  pull_request_review_comment:
+    types:
+      - created
+
+env:
+  APPROVERS_PLUGINS: |-
+    merge
+  APPROVERS: |-
+    kebe7jun
```
Comment (Xunzhuo) [inline .github/workflows/ci-bot.yml:14]: Please add my ID here too :)

**Comment by Xunzhuo** [comment]
> You have to manage the APPROVERS list here. (The current version of [gh-ci-bot](https://github.com/wzshiming/gh-ci-bot/) cannot read from https://github.com/merbridge/merbridge/blob/059c65629628b6b4df73dc2e02038cc1c7b9ef57/CODEOWNERS 😄)

Manage a list manually, I think it is a bit inconvenient, maybe we should try to use Mergify instead, it can assign privileges to a team like @merbridge/infra-maintainers

**Comment by kebe7jun** [comment]
The functionality of this product seems too complex for us now, and we don't seem to need such a complex definition for now. Thanks to paco's suggestion, we'll try this project again later if we need to.

**Comment by kebe7jun** [comment]
we have decided to use mergify bot.

Thank you.

**Pull request closed without merging**